addon/-base-popper-modifier.js

```diff
@@ -65,7 +66,12 @@ export default class PopperModifier extends Modifier {
     return options;
   }
 
-  didReceiveArguments() {
+  modify(element, positionalArgs, namedArgs) {
+    this.primaryElement = element;
+    this.secondaryElement = positionalArgs[0];
+
+    this.popperOptions = this.initPopperOptions(positionalArgs, namedArgs);
```

Comment:
Why is this set on the class instance? It seems to be only used in this method. It seems as if we could refactor to a simple variable. Or did I missed something?

```suggestion
    const popperOptions = this.initPopperOptions(positionalArgs, namedArgs);
```

Reply:
I pretty much just tried to not change too many things, so I kept the style of making options available on the instance.